etc/ci/performance/test_perf.sh

```diff
@@ -12,8 +12,14 @@ TP5N_SOURCE="https://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip"
 TP5N_PATH="page_load_test/tp5n.zip"
 if [[ ! -f "${TP5N_PATH}" ]]; then
     wget "${TP5N_SOURCE}" -O "${TP5N_PATH}"
+else
+    echo "Found existing test cases, skipping download."
```

Comment:
Check for the extracted test cases first, since we can skip downloading the zip file if we already have the extracted copies of the files.